timesketch/lib/analyzers/feature_extraction.py

```diff
@@ -161,6 +193,11 @@ def extract_feature(self, name, config):
         query = config.get('query_string')
         query_dsl = config.get('query_dsl')
         attribute = config.get('attribute')
+        store_type_list = config.get('store_type_list', False)
+        keep_multimatch = config.get('keep_multimatch', False)
+        overwrite_store_as = config.get('overwrite_store_as', True)
+        overwrite_and_merge_store_as = \
```

Comment:
don't use \

rather do

```
overwrite_and_merge_store_as = config.get(
    'overwrite_and_merge_store_as', False)
```

This is more consistent with the rest of the codebase